version

```diff
@@ -1 +1 @@
-0.9.3
+0.9.5
```

Comment:
I expect, that these 2 files should be updated to 0.9.5 as well:
https://github.com/calabash/calabash-android-server/blob/develop/server/integration-tests/calabash-test-suite/Gemfile.lock#L6

https://github.com/calabash/calabash-android-server/blob/develop/server/integration-tests/calabash-test-suite-fast/Gemfile.lock#L6

CI will fail because calabash-android 0.9.5 gem is not released but this is expected.